src/tutorial/runtime/introduction.rst

```diff
@@ -291,60 +275,161 @@ Where Do All The Files Go?
 
          ~/cylc-run/<workflow-name>/log/job/<cycle-point>/<task-name>/<job-submission-num>/
 
-      .. image:: ../img/cylc-gui-view-log.png
+      .. TODO REPLACE THIS IF APPROPRIATE: .. image:: ../img/cylc-gui-view-log.png
```

Comment:
Ditto.

Reply:
I think that this is already commented, and I'm not sure what else needs commenting.